src/us/kbase/workspace/kbase/InitWorkspaceServer.java

```diff
@@ -100,22 +99,22 @@ public boolean isFailed() {
 		private final WorkspaceAdministration wsadmin;
 		private final Types types;
 		private final BasicShockClient linkedShockClient;
-		private final URL handleManagerUrl;
+		private final URL handleServiceUrl;
```

Comment:
I don't think the handle service url is needed here either given my comments on the server class.

Reply:
👍